hq/app/schedule/unrecognised/IamUnrecognisedUsers.scala

```diff
@@ -1,22 +1,20 @@
 package schedule.unrecognised
 
 import com.gu.janus.model.JanusData
-import model.{AwsAccount, CredentialReportDisplay, HumanUser, Tag}
-import utils.attempt.FailedAttempt
+import model.{HumanUser, VulnerableUser}
 
 object IamUnrecognisedUsers {
-  def getJanusUsernames(janusData: JanusData): List[String] =
-    janusData.access.userAccess.keys.toList
+  val USERNAME_TAG_KEY = "GoogleUsername"
 
-  def getHumanUsers(credsReport: Map[AwsAccount, Either[FailedAttempt, CredentialReportDisplay]]): List[HumanUser] =
-    credsReport.values.collect { case Right(report) => report.humanUsers }.flatten.toList
+  def getJanusUsernames(janusData: JanusData): Seq[String] =
+    janusData.access.userAccess.keys.toList
 
-  def filterUnrecognisedIamUsers(iamHumanUsersWithTargetTag: List[HumanUser], targetKey: String, janusUsernames: List[String]): List[HumanUser] =
+  def filterUnrecognisedIamUsers(iamHumanUsersWithTargetTag: Seq[HumanUser], janusUsernames: Seq[String]): Seq[VulnerableUser] =
     iamHumanUsersWithTargetTag.filterNot { iamUser =>
-      val maybeTag = iamUser.tags.find(tag => tag.key == targetKey)
+      val maybeTag = iamUser.tags.find(tag => tag.key == USERNAME_TAG_KEY)
       maybeTag match {
         case Some(tag) => janusUsernames.contains(tag.value) // filter out human users that have tags which match the janus usernames
         case None => true
       }
```

Comment:
```suggestion
      iamUser.tags
        .find(tag => tag.key == USERNAME_TAG_KEY)
        .exists(tag => janusUsernames.contains(tag.value))
```

Reply:
Does that work if the tag doesn't exist (where we want the expression to return true)? Either way, we can update this code in another PR if worthwhile.